packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -118,6 +119,8 @@ async function* loadAllSequencedMessages(
 		lastSeq = firstAvailableDelta - 1;
 	}
 
+	console.log("fetching remaining messages from delta storage");
```

Comment:
I put the two console.log's in this file as I think there's some issue with some assumptions in the subsequent logic where it can get stuck thinking it's waiting for more deltas when in actuality it already has them all. I was considering looking into this later, but for now this output should at least help give people an idea of what fetch-tool is doing.